README.md

```diff
@@ -143,6 +143,7 @@ Supported properties in the `config.*.json` files:
 | `api-url`                 |           | URL of the TranSMART API to connect to. |
 | `api-version`             | `v2`      | TranSMART API version. Only `v2` is supported. |
 | `app-url`                 |           | URL where the Glowing Bear is accessible for the user.|
+| `gb-backend-url`          |           | URL of the Gb-backend application to connect to for cohorts handling. |
```

Comment:
is this to replace api-url? if so, remove api-url completely? or point api-url to gb-backend-url?

Reply:
actually gb-backend is additional application with user queries only. The whole query related functionality (calls starting with '/queries') was moved from transmart to gb-backend, but the other calls will be still handled in transmart. Maybe we could rename the api-url to transmart-url, or something like this, since it's no longer the only API gb is using.

Reply:
this seems to be an effort of better organising/ modularising transmart code, however, from glowing bear perspective, there is just one backend, which is capable of handling all the api calls, separating /queries calls from other calls seems arbitrary, front-end wise. It is better to have a unified frontend also for the reason that glowing bear is intended to be a cohort selection tool not just for transmart, but also for other backends, having two backends for transmart calls is not that sensible and is obstructive for future extension.

Reply:
how much effort does it require for gb-backend to relay non queries calls to transmart? so that gb-backend is the only cohort-selection/export backend for glowingbear

Reply:
never mind, this seems to be too much to change, I am fine with the current choice